openstack/dcs/v1/lifecycle/Get.go

```diff
@@ -0,0 +1,181 @@
+	// Username used for accessing a DCS instance with password authentication.
+	AccessUser string `json:"access_user"`
+	// An indicator of whether public access is enabled for a DCS Redis instance. Options:
+	EnablePublicIp string `json:"enable_publicip"`
+	// ID of the elastic IP address bound to a DCS Redis instance.
+	// The parameter value is null if public access is disabled.
+	PublicIpId string `json:"publicip_id"`
+	// Elastic IP address bound to a DCS Redis instance.
+	// The parameter value is null if public access is disabled.
+	PublicIpAddress string `json:"publicip_address"`
+	// An indicator of whether to enable SSL for public access to a DCS Redis instance.
+	EnableSsl bool `json:"enable_ssl"`
+	// An indicator of whether an upgrade task has been created for a DCS instance.
+	ServiceUpgrade bool `json:"service_upgrade"`
+	// Upgrade task ID.
+	// If the value of service_upgrade is set to true, the value of this parameter is the ID of the upgrade task.
+	// If the value of service_upgrade is set to false, the value of this parameter is empty.
+	ServiceTaskId string `json:"service_task_id"`
+	// Edition of DCS for Redis. Options:
+	// generic: standard edition
+	// libos: high-performance edition
+	ProductType string `json:"product_type"`
+	// CPU architecture. Options: x86_64 and aarch_64.
+	CpuType string `json:"cpu_type"`
+	// Memory type. Options: DRAM and SCM.
+	StorageType string `json:"storage_type"`
+	// DCS instance type. Options:
+	// single: single-node
+	// ha: master/standby
+	// cluster: Redis Cluster
+	// proxy: Proxy Cluster
+	CacheMode string `json:"cache_mode"`
+	// Time when the instance started running. 2022-07-06T09:32:16.502Z
+	LaunchedAt string `json:"launched_at"`
+	// AZ where a cache node resides. The value of this parameter in the response contains an AZ ID.
+	AvailableZones []string `json:"available_zones"`
+	// Subnet ID.
+	SubnetID string `json:"subnet_id"`
+	// Security group ID.
+	SecurityGroupID string `json:"security_group_id"`
+	// Backend address of a cluster instance.
+	BackendAddrs string `json:"backend_addrs"`
+	// Cloud service type code.
+	CloudServiceTypeCode string `json:"cloud_service_type_code"`
+	// Cloud resource type code.
+	CloudResourceTypeCode string `json:"cloud_resource_type_code"`
+	// Product ID.
+	ProductID string `json:"product_id"`
+	// Security group name.
+	SecurityGroupName string `json:"security_group_name"`
+	// Subnet name.
+	SubnetName string `json:"subnet_name"`
+	// Subnet segment.
+	SubnetCIDR string `json:"subnet_cidr"`
+	// Order ID.
+	OrderID string `json:"order_id"`
+	// Backup policy.
+	// This parameter is available for master/standby and cluster DCS instances.
+	InstanceBackupPolicy InstanceBackupPolicy `json:"instance_backup_policy"`
+	// Instance tag key and value.
+	Tags []tags.ResourceTag `json:"tags"`
+	// Product specification code.
+	SpecCode string `json:"spec_code"`
+	// Domain name of the instance.
+	DomainName string `json:"domainName"`
```

Comment:
json string is `domainName` instead of `domain_name` in documentation

Reply:
they are same in latest document
but I chaged it anyway